CI (#4)
sicpa-dlab/peer-did-python: README.md

# Peer DID python API

> ## Information

Reviewer: I would remove the Information header, so that general info is part of the main Peer DID python API header.

> This is a work-in-progress implementation of the Peer DID specification:

Reviewer: I would remove the `work-in-progress` statement. We are considering static support is enough for many cases and can be considered as a valuable MVP.

 - [Peer DID method specification](https://identity.foundation/peer-did-method-spec/)

> This version of API implements only [static layer of support](https://identity.foundation/peer-did-method-spec/#layers-of-support)

Reviewer: ```suggestion
This version of API implements only [static layers of support (1, 2a, 2b)](https://identity.foundation/peer-did-method-spec/#layers-of-support)
```

## Example

Example code:

>     encryption_keys = [PublicKeyAgreement(encoded_value="DmgBSHMqaZiYqwNMEJJuxWzsGGC8jUYADrfSdBrC6L8s",

Reviewer: I think indentation here can be improved:
```
encryption_keys = [
    PublicKeyAgreement(type=PublicKeyTypeAgreement.X25519, encoding_type=EncodingType.BASE58,
                       encoded_value="DmgBSHMqaZiYqwNMEJJuxWzsGGC8jUYADrfSdBrC6L8s")
]
signing_keys = [
    PublicKeyAuthentication(type=PublicKeyTypeAuthentication.ED25519, encoding_type=EncodingType.BASE58,
                            encoded_value="ByHnpUCFb1vAfh9CFZ8ZkmUZguURW8nSw889hy6rD8L7")
]
service = \
'''
    {
            "type": "didcommmessaging",
            "serviceEndpoint": "https://example.com/endpoint",
            "routingKeys": ["did:example:somemediator#somekey"]
    }
'''
peer_did_algo_0 = create_peer_did_numalgo_0(inception_key=signing_keys[0])
peer_did_algo_2 = create_peer_did_numalgo_2(encryption_keys=encryption_keys,
                                            signing_keys=signing_keys,
                                            service=service)
```

                "type": "didcommmessaging",
                "serviceEndpoint": "https://example.com/endpoint",
                "routingKeys": [
                    "did:example:somemediator#somekey"
                ]
            },
            {
                "id": "did:peer:2.Ez6LSpSrLxbAhg2SHwKk7kwpsH7DM7QjFS5iK6qP87eViohud.Vz6MkqRYqQiSgvZQdnBytw86Qbs2ZWUkGv22od935YF4s8M7V.SW3sidCI6ImRtIiwicyI6Imh0dHBzOi8vZXhhbXBsZS5jb20vZW5kcG9pbnQiLCJyIjpbImRpZDpleGFtcGxlOnNvbWVtZWRpYXRvciNzb21la2V5Il19LHsidCI6ImV4YW1wbGUiLCJzIjoiaHR0cHM6Ly9leGFtcGxlLmNvbS9lbmRwb2ludDIiLCJyIjpbImRpZDpleGFtcGxlOnNvbWVtZWRpYXRvciNzb21la2V5MiJdfV0=#example",
                "type": "example",
                "serviceEndpoint": "https://example.com/endpoint2",
                "routingKeys": [
                    "did:example:somemediator#somekey2"
                ]
            }
        ]
    }

## How to contribute
Pull requests are welcome!

> To contribute to this repository you should fork it and do pull request. Please open an issue first to discuss the proposed changes.

Reviewer: ```suggestion
To contribute to this repository you should fork it and do a pull request. Please open an issue first to discuss the proposed changes.
```

> Pull requests will be evaluated by the repository guardians on a schedule and if deemed beneficial will be committed to 

Reviewer: I would remove the first sentence here.

the main branch. Pull requests should have a descriptive name, include the summary of all changes made in the pull request 
description, and include unit tests that provide good coverage of the feature or fix. A Continuous Integration (CI) 
pipeline is executed on all PRs before review and contributors are expected to address all CI issues identified.

### A Continuous Integration (CI) pipeline does the following jobs:

> - Executes all unit tests from pull request.

Reviewer: ```suggestion
- Executes all unit tests from the pull request.
```
